---
subscriber_topic: /actions
movement_publisher_topic: /cmd_vel
dock_publisher_topic: /dock
undock_publisher_topic: /undock
queue_size: 10